docs/reference/docs/delete-by-query.asciidoc

```diff
@@ -229,7 +229,10 @@ include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=stats]
 
 include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=terminate_after]
 
-include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=timeout]
+`timeout`::
+(Optional, <<time-units, time units>>) Time period to wait for
```

Comment:
This `timeout` is unlike other timeouts. It is passed to the underlying bulk requests. See my comment on bulk (and delete).